docs/get-started/alerting.md

```diff
@@ -6,64 +6,161 @@
 Alerting notifies of important or unusual activity in your database environments so that you can identify and resolve problems quickly. When something needs your attention, PMM automatically sends you an alert through your specified contact points.
 
 ## Alert types
-Percona Alerting is powered by Grafana infrastructure. PMM leverages Grafana's advanced alerting capabilities and adds an extra layer of alert templates that simplifies complex alert rules.
 
-Depending on the datasources that you want to query, and the complexity of your required evaluation criteria, PMM enables you to create the following types of alerts: 
+Percona Alerting is powered by Grafana infrastructure. PMM leverages Grafana's advanced alerting capabilities and adds an extra layer of alert templates that simplifies complex alert rules.
```

Comment:
sugg change: Percona Alerting is powered by Grafana infrastructure. **(Percona Alerting)** leverages Grafana's advanced alerting capabilities and **provides pre-configured alert rule templates that** **simplify creating** complex alerting rules.